recipes/liburing/all/test_package/conanfile.py

```diff
@@ -1,15 +1,25 @@
 from conans import ConanFile, CMake, tools
 import os
+import platform
+import re
+
 
 class TestPackageConan(ConanFile):
-    settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
 
     def build(self):
         cmake = CMake(self)
         cmake.configure()
         cmake.build()
 
+    @property
+    def _sufficient_linux_kernel_version(self):
```

Comment:
seems odd to move this to the test_package 🤔

Reply:
The library liburing can be built without any issue, it's using the library that might not be possible.
There could probably be some user/env variable that sets a variable, or a warning in `package_info`.
I moved it to the test package to avoid using any build information in `package_info`.